core/src/main/java/kafka/server/share/DelayedShareFetch.java

```diff
@@ -808,9 +809,18 @@ private void releasePartitionLocksAndAddToActionQueue(Set<TopicIdPartition> topi
         // then we should check if there is a pending share fetch request for the topic-partition and complete it.
         // We add the action to delayed actions queue to avoid an infinite call stack, which could happen if
         // we directly call delayedShareFetchPurgatory.checkAndComplete
-        replicaManager.addToActionQueue(() -> topicIdPartitions.forEach(topicIdPartition ->
+        replicaManager.addToActionQueue(() -> topicIdPartitions.forEach(topicIdPartition -> {
```

Comment:
Could we move the comment above to below this line?

Reply:
Done, and updated the commented with minor change. Frop `topic-partition` => `share-partition`.